kubernetes/configs/dev/commons.sh

```diff
@@ -15,6 +15,7 @@ BACKEND_SERVICE_PORT="8088"
 
 AWS_REGION="eu-central-1"
 PERSISTENCE_QUEUE_NAME="persistence-events.fifo"
+ENVELOPE_QUEUE_NAME="certified-mail.fifo"
```

Comment:
as a common configuration `ENVELOP` could be too generic.
`CERTIFIED_MAIL_QUEUE_NAME` is a viable alternative IMO